CoreFoundation/Parsing.subproj/CFXMLInterface.c

```diff
@@ -22,7 +23,7 @@
 #include <libxml/xpath.h>
 #include <libxml/xpathInternals.h>
 #include <libxml/dict.h>
-#include "CFInternal.h"
```

Comment:
So we don't use CFInternal.h here at all?

Reply:
It wasn't removed, it moved up & uses qualified path. Check line 15: <CoreFoundation/CFInternal.h>

Reply:
Got it, thanks. Misread the diff.